### PR TITLE
Add page with distance records per SAT

### PR DESCRIPTION
**Pull request by phl0**

### CODE REVIEW

**Review by int2001 — APPROVED** [review]
i like. works as designed ;)

File: application/controllers/Distancerecords.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        // Render Page
+        $data['page_title'] = __("Distance Records");
+        $data['distances'] = $this->distancerecords_model->get_records();
+
+        $this->load->view('interface_assets/header', $data);
+        $this->load->view('distancerecords/index');
+        $this->load->view('interface_assets/footer');
+    }
+
+    public function distancerecords() {
```
Comment (int2001) [inline application/controllers/Distancerecords.php:28]: empty?

File: application/controllers/Distancerecords.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    function __construct()
+    {
+        parent::__construct();
+
+        $this->load->model('user_model');
+        if(!$this->user_model->authorize(2)) { $this->session->set_flashdata('error', __("You're not allowed to do that!")); redirect('dashboard'); }
+    }
+
+    public function index()
```
Comment (int2001) [inline application/controllers/Distancerecords.php:14]: Formatting (rest as well)? ;)

**Review by int2001 — CHANGES_REQUESTED** [review]
Cosmetics (Formatting) and empty function

File: application/controllers/Distancerecords.php (hunk 1 above)
Comment (phl0) [reply application/controllers/Distancerecords.php]: fixed.

File: application/controllers/Distancerecords.php (hunk 2 above)
Comment (phl0) [reply application/controllers/Distancerecords.php:14]: done.

**Comment by phl0** [comment]
Issues fixed. Aggree @int2001?

**Review by int2001 — APPROVED** [review]

**Review by AndreasK79 — APPROVED** [review]

**Pull request merged**